contracts/HubPool.sol

```diff
@@ -31,39 +34,41 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         bool isEnabled;
     }
 
-    struct RelayerRefundRequest {
+    struct RefundRequest {
         uint64 requestExpirationTimestamp;
-        uint64 unclaimedPoolRebalanceLeafs;
+        uint64 unclaimedPoolRebalanceLeafCount;
         bytes32 poolRebalanceRoot;
         bytes32 destinationDistributionRoot;
-        mapping(uint256 => uint256) claimedBitMap;
+        uint256 claimedBitMap;
```

Comment:
I think this is a good change, but we've started making some assumptions like "there will never be more than X tokens or Y chains". I want to write these down somewhere. Should we go back to the interface doc and write these down? Or start a known issues and constraints doc to outline this sort of stuff? Thoughts?

I will go ahead and start such a doc, but let me know if you disagree.

Reply:
ok great. ye, you right, we need to write this down! the interface doc is now somewhat out of date though in some regards. our options are to update the interface doc or start a new one.

in either case, I'm going to add some documentation on this.